### DISCUSSION TITLE
Scam NFT issue

### DISCUSSION THREAD

**Discussion by Ogrekin** [sunflower-land/sunflower-land#2878] [Q&A]
I have a simple question about a problem i only have with the wallet i use for my farm...
I get multiple scam nft minutes after i sync or claim daily chest and it's only then.
99.9% of transactions i do with that wallet is sync, claim  and few withdraws nothing more than that
It feels like there are some tracker on sunflower-land accounts that activate multiple drops i do anything.

I just keep ignoring them but it's starting to get annoying to get multiple of notifications about receiving nfts
and sometimes they can keep coming hours after a claim or sync...
Is there anything i can do other than change wallet?? It's just crazy i only have it in a wallet i barely use..

There are coming so many different tools now for all kinds of stuff related to web3 so it would be great if 
there was some type of firewall that can block it, so if anyone knows any tool i will happily try them..

![wl2](https://github.com/sunflower-land/sunflower-land/assets/120697821/6f863092-82f1-473a-9e7b-78193e7a13c6)
![wl3](https://github.com/sunflower-land/sunflower-land/assets/120697821/362c43c1-8933-4e33-bb86-d4dda176717e)

**Comment by eliasSFL** [comment]
First of all this isn't really an SFL issue, this issue is persistent in the blockchain ecosystem. As long as your wallet address is public and has done 1 transaction before, scammers will send scam nfts to your wallet to try to make you interact with their contract. 
All you can do is ignore these transactions. you can't get rid of it, but you can at least hide them in opensea
